src/back/gql/models/users/types.js

```diff
@@ -23,4 +28,20 @@ export const UserType = new GraphQLObjectType({
   },
 });
 
+export const CreateUserInputType = new GraphQLInputObjectType({
+  name: 'CreateUserInputType',
+  description: 'Input needed to create an User record',
+  fields: {
```

Comment:
What about an id field?

Reply:
ID is not input by the end-user (you guys), ID is auto-generated. Check here:
https://github.com/jrojas28/react-reddit/pull/8/files#diff-e8c8f9249f1e695d440432f8459f1501R63

I assign an ID automatically :)